resources/lib/functions.py

```diff
@@ -588,7 +588,10 @@ def processDirectory(results, progress, params):
         name_format = settings.getSetting(name_format)
 
     dirItems = []
-    result = results.get("Items")
+    try:
+        result = results.get("Items")
```

Comment:
That doesn't work because result = results.get("Items") throws an error.  Changed it to this:
```

    if results is None:
        result = []
    if "Items" in results:
        result = results.get("Items") 
    else: 
        result = results
```